core/src/main/scala/ai/h2o/sparkling/backend/shared/SharedBackendConf.scala

```diff
@@ -122,6 +123,7 @@ trait SharedBackendConf {
 
   def ignoreSparkPublicDNS: Boolean = sparkConf.getBoolean(PROP_CLIENT_IGNORE_SPARK_PUBLIC_DNS._1, PROP_CLIENT_IGNORE_SPARK_PUBLIC_DNS._2)
 
+  @DeprecatedMethod(version="3.30")
   def clientWebEnabled: Boolean = sparkConf.getBoolean(PROP_CLIENT_ENABLE_WEB._1, PROP_CLIENT_ENABLE_WEB._2)
```

Comment:
Can't we return always true and remove the property?

Reply:
Yup!

Reply:
Yup!